### ISSUE TITLE
[brace-style] Allow '1tbs' to have comment line before `else`

### ISSUE THREAD

**Issue by SrBrahma** [eslint/eslint#14871]
**What rule do you want to change?** [`brace-style`](https://eslint.org/docs/rules/brace-style), '1tbs' option

**Does this change cause the rule to produce more or fewer warnings?** Ain't sure if this an yes/no question or an objective one. It would reduce the warnings.

**How will the change be implemented? (New option, new default behavior, etc.)?** New option

**About my request**: I like the '1tbs', it's usually the best choice for me, but it doesn't allow comments on a line before an `else` block. I am forced to write the comment next to the `else` keyword or inside its block. Sometimes a comment before the `else` is very good for code explanation.

**Please provide some example code that this change will affect:**

```js
/*eslint brace-style: ["error", "1tbs"] */
// Ok case
if (true) {
...
} else { // ~ 'Some explanatory comment about else block.'
...
}

// Not allowed, warn: `Closing curly brace does not appear on the same line as the subsequent block.`
if (true) {
...
}
// ~ 'Some explanatory comment about else block.' It isn't allowed.
else {
...
}

```

**What does the rule currently do for this code?** If '1tbs', it doesn't allow `else` after a comment line.

**What will the rule do after it's changed?** If '1tbs', allow `else` if proceeding a comment line.

**Are you willing to submit a pull request to implement this change?** No, I don't have any knowledge on eslint internals.

**Comment by nzakas** [comment]
Thanks for the input. As [stated in the README](https://github.com/eslint/eslint#stylistic-rule-updates), we actually aren't making any further updates to rules that enforce stylistic preferences. If you have a strong preference for such a change, you can always copy the existing rule into a custom rule and make it behave the way you want.

**Comment by SrBrahma** [comment]
I understand. I just saw https://github.com/eslint/eslint/blob/master/lib/rules/brace-style.js, and with some tinkering seems to be doable for me. Even if I get it working properly, it wouldn't be possible to make a PR to eslint, right? I would have to create my own package for it, right?
